connection.go

```diff
@@ -80,12 +82,12 @@ func (d defaultLogger) Report(event ConnLogKind, conn *Connection, v ...interfac
 	}
 }
 
-// Connection is a handle to Tarantool.
+// Connection is a handle with a single connection to Tarantool instance.
 //
 // It is created and configured with Connect function, and could not be
 // reconfigured later.
 //
-// It is could be "Connected", "Disconnected", and "Closed".
+// It could be in "Connected", "Disconnected", and "Closed" states.
```

Comment:
```suggestion
// It could be in "Connected", "Disconnected", or "Closed" state.
```

Reply:
```diff
--- a/connection.go
+++ b/connection.go
@@ -82,22 +82,23 @@ func (d defaultLogger) Report(event ConnLogKind, conn *Connection, v ...interfac
        }
 }
 
-// Connection is a handle with a single connection to Tarantool instance.
+// Connection is a handle with a single connection to a Tarantool instance.
 //
 // It is created and configured with Connect function, and could not be
 // reconfigured later.
 //
-// It could be in "Connected", "Disconnected", and "Closed" states.
+// Connection could be in three possible states:
 //
-// When "Connected" it sends queries to Tarantool.
+// - in "Connected" state it sends queries to Tarantool.
 //
-// When "Disconnected" it rejects queries with ClientError{Code: ErrConnectionNotReady}
+// - in "Disconnected" state it rejects queries with ClientError{Code:
+// ErrConnectionNotReady}
 //
-// When "Closed" it rejects queries with ClientError{Code: ErrConnectionClosed}
-//
-// Connection could become "Closed" when Connection.Close() method called,
-// or when Tarantool disconnected and Reconnect pause is not specified or
-// MaxReconnects is specified and MaxReconnect reconnect attempts already performed.
+// - in "Closed" state it rejects queries with ClientError{Code:
+// ErrConnectionClosed}. Connection could become "Closed" when
+// Connection.Close() method called, or when Tarantool disconnected and
+// Reconnect pause is not specified or MaxReconnects is specified and
+// MaxReconnect reconnect attempts already performed.
 //
 // You may perform data manipulation operation by calling its methods:
 // Call*, Insert*, Replace*, Update*, Upsert*, Call*, Eval*.
```

Rendered:

![image](https://user-images.githubusercontent.com/1151557/163359305-653aa70c-acd1-4e52-b917-e9ca861b724a.png)

Reply:
If it ends with a dot, it must start with a capital letter (`In "Connected"` etc)

Reply:
Fixed.

```diff
--- a/connection.go
+++ b/connection.go
@@ -89,12 +89,12 @@ func (d defaultLogger) Report(event ConnLogKind, conn *Connection, v ...interfac
 //
 // Connection could be in three possible states:
 //
-// - in "Connected" state it sends queries to Tarantool.
+// - In "Connected" state it sends queries to Tarantool.
 //
-// - in "Disconnected" state it rejects queries with ClientError{Code:
+// - In "Disconnected" state it rejects queries with ClientError{Code:
 // ErrConnectionNotReady}
 //
-// - in "Closed" state it rejects queries with ClientError{Code:
+// - In "Closed" state it rejects queries with ClientError{Code:
 // ErrConnectionClosed}. Connection could become "Closed" when
 // Connection.Close() method called, or when Tarantool disconnected and
 // Reconnect pause is not specified or MaxReconnects is specified and
```